docs/source/api/plugin/subscription-callback.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: "API Reference: Subscription callback plugin"
+api_reference: true
+---
```

Comment:
```suggestion
minVersion: 4.9.0
---
```

Reply:
Yesss, thanks for suggesting use of the new component @dariuszkuc ! ❤️